keps/sig-instrumentation/0035-20190131-event-series.md

```diff
@@ -0,0 +1,59 @@
+---
+kep-number: 35
+title: Event series API
+authors:
+  - "@gmarek"
+owning-sig: sig-instrumentation
+participating-sigs:
+  - sig-instrumentation
+  - sig-scalability
+  - sig-architecture
+reviewers:
+  - "@wojtekt"
+  - "@bgrant0607"
+approvers:
+  - "@wojtekt"
+  - "@bgrant0607"
+editor: TBD
+creation-date: 2019-1-31
+last-updated: 2019-1-31
+status: implementable
+see-also:
+  - n/a
+replaces:
+  - n/a
+superseded-by:
+  - n/a
+---
+
+# Title
+
+Event series API
+
+## Table of Contents
+
+* [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+
+
+This KEP wraps the already merged design doc under [kubernetes/community](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/instrumentation/events-redesign.md)
+
+## Graduation Criteria
+
+Beta:
+
+- noticeable performance improvement
+- test coverage for edge-cases
+- switched all the controllers to use the new Event API
```

Comment:
Don't we need the API to be beta BEFORE we can switch all the controllers to it, or are we going to use feature gates to select between the original and new APIs?

Reply:
The API is already in beta IIRC

Reply:
Yeah - I don't understand this point.
We first need to promote (well, we need to have it) the implementation of the API to Beta, and only then start switching existing stuff to the new API.